lib/inc/class-wp-rest-coauthors-authorposts.php

```diff
@@ -84,6 +84,19 @@ public function get_items( $request ) {
 
 		$authors = array();
 
+		// If number and offset are set in the $request, use them,
+		// otherwise default to show 100
+		$number = ! empty( $request['number'] && 100 >= $request['number'] ) ? $request['number'] : 100;
```

Comment:
If `$request['number']` is greater than 100 this will set to that value but what if I want less than 100, like 10 or 20?